metadata.rb

```diff
@@ -4,7 +4,7 @@
 license          'Apache 2.0'
 description      'provides LWRPs for interaction with AWS'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          '2.0.1'
+version          '2.0.2'
```

Comment:
maybe add the chef version it supports here as well now?

Reply:
It is already in there (last row, just under the unexpanded bit).